### PR TITLE
Second phase completed

### PR DESCRIPTION
**Pull request by Hombre2014**
All the requirements from the step 2 - interactive list, have been implemented:
- Add a new JavaScript files and import it as a module:
  - it will contain methods related to the status updates (`completed`: `true` / `false`).
- Add event listener to the checkbox (`change`).
- Update items object's value for `completed` key upon user actions.
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @Hombre2014 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -65,7 +73,7 @@ To get a local copy up and running follow these simple example steps.
 
 ### Deployment
 
-- All the files necessary for deployment are in the /dist folder
+- All the files necessary for deployment are in the `/dist` folder
```
Comment (mikenath223) [inline README.md:76]: - [ ] In `src/status.js` kindly change all regular functions to es6 arrow function syntax following the JavaScript best practices guide.

Comment (Hombre2014) [reply README.md:76]: I really do not understand your first comment about the README file. I do have
- All the files necessary for deployment are in the `/dist` folder stated there! Please, clarify!
For your second comment I would say:
- I used arrow functions where I thought is appropriate and useful. For example in `checkButton` function. There is no specific requirements to have all my regular functions to be arrow functions! Please, have a look at project requirements from my [README.md](https://github.com/Hombre2014/ToDoList/tree/interactive#readme) file (step 2).
- In [Java Best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) article from Microverse there is no rule to have all regular functions as arrow functions.
- In [code clean](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices) article cited above also there is nothing stating that you should use only arrow functions!
- I am repeating the week and this same branch (code) have been approved last week. You can see it [here](https://github.com/Hombre2014/ToDoList/pull/2#pullrequestreview-821188860).
So, I do not really understand your rejection. These might be recommendations and optional changes, but rejection? Please, reply.

**Review by mikethreels — APPROVED** [review]
Hi Yuriy , 

Apologies for the misunderstanding in the previous review, usage of ES6 arrow functions is indeed not a requirement.

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**